resources/qml/application.qml

```diff
@@ -138,40 +76,159 @@ ApplicationWindow {
+                    spacing: 20
+
+                    Item {
+                        width: 1
+                    }
+                    Waveform {
                         opacity: .9
                         context: defaultContext
                     }
-                    Waveform {
-                        width: 500
+                    BeatIndicator {
                         opacity: .9
                         context: defaultContext
                     }
                     Spectrum {
-                        width: 500
                         opacity: .9
                         context: defaultContext
                     }
+                    Item {
+                        Layout.fillWidth: true
+                    }
+                    RadianceButton {
+                        Item {
+                            id: contentItem
+                            property real oversample: 4
+                            anchors.fill: parent
+                            anchors.margins: 3
+                            scale: 1. / oversample
+                            Image {
+                                id: image
+                                anchors.centerIn: parent
+                                source: "../graphics/gear.svg"
+                                fillMode: Image.PreserveAspectFit
+                                visible: false
+                                width: parent.width * contentItem.oversample
+                                height: parent.height * contentItem.oversample
+                                sourceSize.height: 128
+                            }
+                            ColorOverlay {
+                                anchors.fill: image
+                                source: image
+                                color: RadianceStyle.tileLineColor
+                            }
+                        }
+                        text: ""
+                        onClicked: settingsWidget.toggle()
+                        implicitHeight: 25
+                        implicitWidth: implicitHeight
+                        colorDark: RadianceStyle.mainBackgroundColor
+                    }
+                    Item {
+                        width: 1
+                    }
                 }
-                RowLayout {
+                SplitView {
+                    id: splitView
+                    property real openWidth: 150
+                    Layout.fillHeight: true
+                    Layout.fillWidth: true
+
                     LibraryWidget {
                         id: libraryWidget
                         graph: graph;
                         registry: registry
                         context: defaultContext
-                        width: 200
+                        implicitWidth: splitView.openWidth
+                        clip: true
                         
                         Layout.fillHeight: true
+
+                        Rectangle {
```

Comment:
The library should have a button to open/close it (like settings does), and the search textbox should probably match the styling of the textboxes in the settings pane

Reply:
I changed the search box styling, I'm torn on the library button. The library will never collapse unless you use : to search, and you can always pull it back open with the mouse.

It's also sort of hard to do, the slidey widget I'm using is deprecated so adding a button will probably mean losing the adjustability.